CommandHelper/LocalPackages/commands/chars/donnatroyrebirth.yml

```diff
@@ -1,10 +1,11 @@
-maxHearts: 80
+maxHearts: 40
 buffs:
-- /effect <plr> 5 9999 8 true
+- /effect <plr> 5 9999 15 true
 - /effect <plr> 10 9999 2 true
-- /effect <plr> 11 9999 2 true
+- /effect <plr> 11 9999 3 true
 - /give <plr> minecraft:iron_sword 1 16:40 name:&0D&7on&0na_T&7ro&0y&7(&0R&7)&0's_&7Sw&0or&7d
 - /give <plr> minecraft:shield 1 durability:10 mending:1 basecolor:16351261 name:&0D&7on&0na_T&7ro&0y&7(&0R&7)&0's_&7Sh&0ie&7ld
+- /give <plr> minecraft:lead 1 sharpness:5 name:&0D&7on&0na_T&7ro&0y&7(&0R&7)&0's_&7la&0ss&7o
```

Comment:
This shield is the wrong kind. You need to check out wonder woman's shield or captain america;s shield on how to properly spawn one. The colors dont show up if you do it this way.